packages/palette_generator/example/pubspec.yaml

```diff
@@ -1,9 +1,10 @@
 name: image_colors
 description: A simple example of how to use the PaletteGenerator to load the palette from an image file.
-
+publish_to: none
 version: 0.1.0
 
-publish_to: none
+environment:
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
```

Comment:
Oops, missed one thing: `cupertino_icons` should be 1.0.0 (or removed if it's not used)

Reply:
I think it can be removed.